plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.xml

```diff
@@ -2,6 +2,67 @@
 <?eclipse version="3.4"?>
 <plugin>
 
+  <extension point="org.eclipse.ui.commands">
+    <command
+      description="%deployStandardCommandDescription"
+      id="com.google.cloud.tools.eclipse.appengine.deploy"
+      name="%deployStandardCommandName">
+    </command>
+  </extension>
+
+  <extension point="org.eclipse.ui.menus">
+    <menuContribution
+      allPopups="false"
+      locationURI="popup:org.eclipse.ui.popup.any?after=additions">
+      <command
+        commandId="com.google.cloud.tools.eclipse.appengine.deploy"
+        label="%deployStandardMenuLabel"
+        style="push">
+        <visibleWhen checkEnabled="true" />
+      </command>
+    </menuContribution>
+  </extension>
+
+  <extension point="org.eclipse.ui.handlers">
+    <handler
+      class="com.google.cloud.tools.eclipse.appengine.deploy.ui.standard.StandardDeployCommandHandler"
+      commandId="com.google.cloud.tools.eclipse.appengine.deploy">
+      <enabledWhen>
+        <and>
+          <count value="1" />
+          <iterate>
+            <reference definitionId="com.google.cloud.tools.eclipse.appengine.onlyInGCPProjects" />
+          </iterate>
+        </and>
+      </enabledWhen>
+    </handler>
+  </extension>
+
+  <extension
+    point="org.eclipse.ui.console.consolePatternMatchListeners">
+    <consolePatternMatchListener
+      class="com.google.cloud.tools.eclipse.ui.util.console.PatternToHyperlinkConverter"
+      id="com.google.cloud.tools.eclipse.appengine.deploy.consolePatternMatchListener"
+      regex="https://.+\.appspot\.com">
```

Comment:
I agree, that it's too broad, however I'd only restrict it to only non-whitespace. I don't want to add a more stricter logic as then it needs to be maintained should the valid format change. I think we can rely on the fact, that if something starts with `https://` and ends with `appspot.com` and appears in the output of the deploy process, it is most likely a valid URL.